concent_api/core/payments/report.py

```diff
@@ -0,0 +1,759 @@
+from datetime import datetime
+from enum import Enum
+from enum import unique
+from typing import Any
+from typing import List
+from typing import NamedTuple
+from typing import Tuple
+
+from django.conf import settings
+from django.db.models import QuerySet
+
+from golem_sci.events import CoverAdditionalVerificationEvent
+from golem_sci.events import ForcedPaymentEvent
+from golem_sci.events import ForcedSubtaskPaymentEvent
+
+from common.constants import ConcentUseCase
+from common.helpers import deserialize_message
+from common.helpers import get_current_utc_timestamp
+from common.helpers import parse_datetime_to_timestamp
+from core.models import Client
+from core.models import DepositClaim
+from core.models import PendingResponse
+from core.models import Subtask
+from core.payments import service
+from core.payments.backends.sci_backend import TransactionType
+
+
+@unique
+class MatchingError(Enum):
+    MESSAGE_NOT_MATCHED_WITH_ANY_PAYMENT = \
+        'message_not_matched_with_any_payment'
```

Comment:
We don't have a line length limit. I'd suggest not breaking lines like this.

Reply:
:+1: done